packages/cli/README.md

```diff
@@ -128,7 +140,7 @@ USAGE
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Private Key
```

Comment:
ditto: r/o command shouldn't need --key

Reply:
See above, keeping this as it does add user benefit.